### DISCUSSION TITLE
OMI glTF Working Group Meeting 2022-04-07

### DISCUSSION THREAD

**Discussion by robertlong** [omigroup/gltf-extensions#77] [General]
# OMI glTF Working Group Meeting 4/7/2022

This meeting will be on 4/7/2022 at 10:30 PM (UTC) / 2:30 PM (PST) in the [OMI Discord](https://discord.gg/NJtT9grz5E) within the #omi-weekly-meeting channel. During the meeting, we will be using the #omi-gltf-extensions channel to manage a speaker queue, post links, and for any sidebar discussions.

To be notified of this meeting and others, subscribe to the [OMI Meetings and Events Calendar](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t) or add yourself to the @omi-gltf-subgroup role in the #roles channel of the OMI Discord.

## Agenda:

- KHR_audio feedback
  - https://github.com/KhronosGroup/glTF/pull/2137
- OMI_collider
  - https://github.com/omigroup/gltf-extensions/pull/63
  - https://github.com/KhronosGroup/glTF/pull/2087
- Avatar Animation Retargeting
  - https://github.com/omigroup/gltf-extensions/issues/44
- Visibility extension proposal
  - https://github.com/omigroup/gltf-extensions/discussions/72#discussioncomment-2383988

To propose another item for the agenda, comment below (preferably before the meeting).

**Comment by antpb** [comment]
https://github.com/omigroup/omi-blender-gltf/issues/8

**Comment by robertlong** [comment]
# Notes
- KHR_audio
  - Animation driven audio playback
  - Random function for changing more generalized properties, not just clip randomization
  - Multiple audio emitters on a single node handled by multiple child nodes or a future audio mixer extension where an audio source is a mixer that combines multiple audio sources
  - Update existing examples  / implementations (three-omi)
    - @antpb to look into updating three-omi
    - @robertlong to update gltf-validator and omi-gltf-transform
  - https://github.com/omigroup/omi-blender-gltf/issues/8
    - Need to change the output path to only include filename + extension @antpb to look into the after three-omi
- OMI_collider
  - Need to specify what multiple mesh.primitives means for a collider? Meshes are referenced by both hulls and trimesh collision shapes. Perhaps this is analogous to compound primitive colliders?
  -  MAXAR_collision_geometry only allows for mesh referenced colliders. Need to sync up with them to see if we can unify our extensions. Theirs is likely a subset of ours
